src/main/scala/uk/gov/nationalarchives/omega/api/repository/OmegaRepository.scala

```diff
@@ -21,19 +21,19 @@
 
 package uk.gov.nationalarchives.omega.api.repository
 
-import org.apache.jena.query.QuerySolution
+import org.apache.jena.query.{ Query, QuerySolution }
 import org.phenoscape.sparql.FromQuerySolution
 import uk.gov.nationalarchives.omega.api.connectors.SparqlEndpointConnector
 import uk.gov.nationalarchives.omega.api.messages.reply.LegalStatus
 import uk.gov.nationalarchives.omega.api.repository.model.AgentEntity
 
 import scala.util.Try
 
-class OmegaRepository(sparqlConnector: SparqlEndpointConnector) extends AbstractRepository {
+class OmegaRepository(sparqlConnector: SparqlEndpointConnector) extends AbstractRepository with RepositoryUtils {
 
   private val sparqlResourceDir = "sparql"
   private val getLegalStatusSummarySparqlResource = s"/$sparqlResourceDir/select-legal-status-summaries.rq"
```

Comment:
should this be renamed to `selectLegalStatusSummarySparqlResource`

Reply:
Renamed.